### PR TITLE
T21837: port to freedesktop.org runtime

### PR DESCRIPTION
**Pull request by ramcq**
Take away the automake/debian stuff and instead ship a flatpak-builder .json which builds the flatpak directly using the freedesktop.org runtime. The icons are from upstream, and the appdata/desktop from the existing Endless Flatpak with some tweaks inspired by the upstream appdata. The resulting flatpak is very similar to what we had before, and can be run by the OS helper in exactly the same way.

https://phabricator.endlessm.com/T21837

### CODE REVIEW

File: com.google.Chrome.desktop [hunk 1]
```diff
@@ -0,0 +1,34 @@
+[Desktop Entry]
+Name[hi]=Google Chrome
```
Comment (cosimoc) [inline com.google.Chrome.desktop]: Any chance you can sort the file?

File: com.google.Chrome.desktop [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+Name[pt_BR]=Google Chrome
+Comment[pt_BR]=Navegador popular do Google
+Name[es]=Google Chrome
+Comment[id]=Peramban web populer dari Google
+Name[ar]=Google Chrome
+Exec=/app/bin/eos-google-chrome-app
+Version=1.0
+Comment[vi]=Trình duyệt web phổ biến nhất của Google
+Type=Application
+Icon=com.google.Chrome
```
Comment (cosimoc) [inline com.google.Chrome.desktop]: Can you also add our custom icon to the EndlessOS flatpak icon theme extension?

File: com.google.Chrome.json [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+            "buildsystem": "simple",
+            "sources": [
+                {
+                    "type": "extra-data",
+                    "filename": "chrome.deb",
+                    "only-arches": [ "x86_64" ],
+                    "url": "http://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-stable/google-chrome-stable_65.0.3325.181-1_amd64.deb",
+                    "sha256": "2668878e7f9eff2a2ffbf2c4393b783e3d67da599b5ff94714ef1fa548e22464",
+                    "size": 52132868,
+                    "x-checker-data": {
```
Comment (cosimoc) [inline com.google.Chrome.json]: Curious, is this a standard piece of metadata now?

Comment (joaquimrocha) [reply com.google.Chrome.json]: Nope. That's my checker's script metadata. But AFAICS it doesn't generate any warning from `flatpak-builder`.
I really need to spend some time trying to push the use of the script in Flathub...

File: com.google.Chrome.desktop (hunk 2 above)
Comment (ramcq) [reply com.google.Chrome.desktop]: https://github.com/endlessm/eos-theme/pull/298

File: com.google.Chrome.desktop (hunk 1 above)
Comment (ramcq) [reply com.google.Chrome.desktop]: Pushed.

**Comment by cosimoc** [comment]
LGTM; @ramcq not sure if the other Jenkins bits are required before this can be merged. But this looks good to me, so feel free to merge when convenient.

**Pull request merged**